doc/user/pim.rst

```diff
@@ -391,6 +391,16 @@ is in a vrf, enter the interface command with the vrf keyword at the end.
    If you enter a value smaller than 60 seconds be aware that this can and
    will affect convergence at scale.
 
+.. clicmd:: ip pim assert-interval (1000-86400000)
+
+   Modify the PIM assert interval in milliseconds on this interface
```

Comment:
Given that the code specifically calls out that the assert interval should not be changed as per RFC, should the same warning be placed in the documentation for the operator?